src/views/Main/CourseContainer/SolveCheckButton/SolveCheckButton.tsx

```
@@ -5,7 +5,7 @@ import { toast } from 'sonner';
 
 const SolveCheckButton = () => {
   const { show } = useConfetti();
-  const { todayProblem, checkTodayProblem, isSolvingCheckLoading } = useUserCourses();
+  const { todayProblem, checkTodayProblem, isSolvingCheckLoading, loadUserCourses} = useUserCourses();
```

ESLint

[error] Replace `}·=` with `·}·=⏎···`

(prettier/prettier)